### PR TITLE
zh: improve the doc of tidb-lightning

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Fix https://github.com/pingcap/docs-tidb-operator/issues/1213

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/restore-from-s3.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-s3/']
 
 # 使用 TiDB Lightning 恢复 S3 兼容存储上的备份数据
 
-本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。底层通过使用 [TiDB Lightning](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来恢复数据。
+本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。
 
-本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来恢复数据。
```
Comment (csuzhangxc) [inline zh/restore-from-s3.md:11]: 要链接到 `https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview` 这个吗？

File: zh/restore-from-s3.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-s3/']
...
+
+目前，TiDB Lightning 支持 `Importer-backend`, `Local-backend` 及 `TiDB-backend` 三种后端，对于这三种后端的区别和选择，可以参考[TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参考[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```
Comment (csuzhangxc) [inline zh/restore-from-s3.md:13]:
```suggestion
目前，TiDB Lightning 支持 `Importer-backend`, `Local-backend` 及 `TiDB-backend` 三种后端，对于这三种后端的区别和选择，可以参考 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参考[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```

File: zh/restore-from-s3.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/restore-from-s3.md]: Fixed.

File: zh/restore-from-s3.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/restore-from-s3.md]: Fixed.

File: zh/restore-data-using-tidb-lightning.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ TiDB Lightning 包含两个组件：tidb-lightning 和 tikv-importer。在 Kuber
 helm inspect values pingcap/tidb-lightning --version=${chart_version} > tidb-lightning-values.yaml
 ```
 
-根据需要配置 TiDB Lightning 所使用的后端 `backend`，即将 `values.yaml` 中的 `backend` 设置为 `importer`, `local` 或 `tidb`。
+根据需要配置 TiDB Lightning 所使用的后端 `backend`，即将 `values.yaml` 中的 `backend` 设置为 `importer`，`local` 或 `tidb`。
```
Comment (qiancai) [inline zh/restore-data-using-tidb-lightning.md:95]:
```suggestion
根据需要配置 TiDB Lightning 所使用的后端 `backend`，即将 `values.yaml` 中的 `backend` 设置为 `importer`、`local` 、或 `tidb` 中的一个。
```

File: zh/restore-from-gcs.md [hunk 4]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-gcs/']
 
 # 使用 TiDB Lightning 恢复 GCS 上的备份数据
 
-本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。底层通过使用 [TiDB Lightning](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来进行集群恢复。
+本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。
 
-本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview) 来恢复数据。
+
+目前，TiDB Lightning 支持 `Importer-backend`，`Local-backend` 及 `TiDB-backend` 三种后端，对于这三种后端的区别和选择，可以参考 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参考[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```
Comment (qiancai) [inline zh/restore-from-gcs.md:13]:
```suggestion
目前，TiDB Lightning 支持 `Importer-backend`、`Local-backend` 、及 `TiDB-backend` 三种后端，关于这三种后端的区别和选择，请参阅 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参阅[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```

File: zh/restore-from-s3.md [hunk 5]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-s3/']
 
 # 使用 TiDB Lightning 恢复 S3 兼容存储上的备份数据
 
-本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。底层通过使用 [TiDB Lightning](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来恢复数据。
+本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。
 
-本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview) 来恢复数据。
+
+目前，TiDB Lightning 支持 `Importer-backend`，`Local-backend` 及 `TiDB-backend` 三种后端，对于这三种后端的区别和选择，可以参考 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参考[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```
Comment (qiancai) [inline zh/restore-from-s3.md:13]:
```suggestion
目前，TiDB Lightning 支持 `Importer-backend`，`Local-backend` 、及 `TiDB-backend` 三种后端，关于这三种后端的区别和选择，请参阅 [TiDB Lightning 文档](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends)。如果要使用 `Importer-backend` 或者 `Local-backend` 导入数据，请参考[使用 TiDB Lightning 导入集群数据](restore-data-using-tidb-lightning.md)。
```

File: zh/restore-from-s3.md [hunk 6]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-s3/']
 
 # 使用 TiDB Lightning 恢复 S3 兼容存储上的备份数据
 
-本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。底层通过使用 [TiDB Lightning](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来恢复数据。
+本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。
 
-本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview) 来恢复数据。
```
Comment (qiancai) [inline zh/restore-from-s3.md:11]:
```suggestion
本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends#tidb-lightning-tidb-backend) 来恢复数据。
```

File: zh/restore-from-gcs.md [hunk 7]
```diff
@@ -6,9 +6,11 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/restore-from-gcs/']
 
 # 使用 TiDB Lightning 恢复 GCS 上的备份数据
 
-本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。底层通过使用 [TiDB Lightning](https://pingcap.com/docs/stable/how-to/get-started/tidb-lightning/#tidb-lightning-tutorial) 来进行集群恢复。
+本文描述了将 Kubernetes 上通过 TiDB Operator 备份的数据恢复到 TiDB 集群的操作过程。
 
-本文使用的恢复方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview) 来恢复数据。
```
Comment (qiancai) [inline zh/restore-from-gcs.md:11]:
```suggestion
本文使用的恢复方式基于 TiDB Operator v1.1 及以上的 CustomResourceDefinition (CRD) 实现，底层通过使用 [TiDB Lightning TiDB-backend](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends#tidb-lightning-tidb-backend) 来恢复数据。
```

File: zh/restore-data-using-tidb-lightning.md [hunk 8]
```diff
@@ -232,21 +232,21 @@ tidb-lightning Helm chart 支持恢复本地或远程的备份数据。
 
 部署 TiDB Lightning 的方式根据不同的权限授予方式及存储方式，有不同的情况。
 
-+ 使用 Amazon S3 AccessKey 和 SecretKey 权限授予方式，或者使用 Ceph，GCS 作为存储后端时，运行以下命令部署 TiDB Lightning：
++ 对于[本地模式](#本地模式)、[Ad hoc 模式](#ad-hoc-模式)和使用 Amazon S3 AccessKey 和 SecretKey 权限授予方式，或者使用 Ceph，GCS 作为存储后端的[远程模式](#远程模式)时，运行以下命令部署 TiDB Lightning：
```
Comment (qiancai) [inline zh/restore-data-using-tidb-lightning.md:235]:
```suggestion
+ 对于[本地模式](#本地模式)、[Ad hoc 模式](#ad-hoc-模式)、[远程模式](#远程模式)（需要是符合以下三个条件之一的远程模式：使用 Amazon S3 AccessKey 和 SecretKey 权限授予方式、使用 Ceph 作为存储后端、使用 GCS 作为存储后端），运行以下命令部署 TiDB Lightning：
```

**Review by qiancai — COMMENTED** [review]
@DanielZhangQD PTAL

**Review by qiancai — APPROVED** [review]

**Pull request merged**